libs/ui/elements/src/lib/downloads-list/downloads-list.component.ts

```diff
@@ -94,6 +124,7 @@ export class DownloadsListComponent {
     if (this.activeFilterFormats.length === 0) {
       this.activeFilterFormats = ['all']
     }
+    this.forceRefreshPagination()
```

Comment:
This absolutely shouldn't be here; pagination is definitely not a concern of this component!